.github/workflows/main.yml

```diff
@@ -12,6 +12,7 @@ jobs:
     runs-on: ubuntu-20.04
 
     strategy:
+      fail-fast: false
```

Comment:
We probably ought to leave this as-is... not strictly related to the rest of the PR, right?

```suggestion
```